api/nodes/serializers.py

```diff
@@ -4,7 +4,22 @@
 from framework.auth.core import Auth
 from rest_framework import exceptions
 from api.base.serializers import JSONAPISerializer, LinksField, Link, WaterbutlerLink
+from ast import literal_eval
 
+class NodeTag(object):
+    def __init__(self, tag):
+        assert object is not None
```

Comment:
What is the purpose of this? `object` is a built-in class; it will never be `None`.